src/main/js/controls/PairedResult/PairedResult.js

```diff
@@ -191,17 +192,43 @@ class PairedResult extends GraphContent {
      */
     resize(graph) {
         if (utils.notEmpty(this.dataTarget.regions)) {
-            // If graph has more than 1 content then we dont load the regions
+            // If graph has more than 1 content, we compare the regions if they are identical and hide if even atleast one of them is not.
+            const regionList = this.dataTarget.regions;
```

Comment:
Move this into line 199 so that it can only be created when there are multiple content.

Reply:
ok i will move the line to 199.